book/src/SUMMARY.md

```diff
@@ -0,0 +1,30 @@
+# Summary
+
+[Overview](overview.md)
+
+# Goose
+
+- [Getting Started](getting-started.md)
+- [Simple Example](simple-example.md)
+- [Scheduling GooseTaskSets](scheduling-goosetasksets.md)
+- [Defaults](defaults.md)
+- [Controlling Running Goose Load Test](controlling-running-goose-load-test.md)
+- [Throttling Requests](throttling-requests.md)
+- [Logging Load Test Errors](logging-load-test-errors.md)
+- [Logging Load Test Requests](logging-load-test-requests.md)
+- [Logging Load Test Tasks](logging-load-test-tasks.md)
+- [Load Test Debug Logging](load-test-debug-logging.md)
+- [Coordinated Omission Mitigation](coordinated-omission-mitigation.md)
+
+# Gaggle
```

Comment:
Do not add/change the structure with this first PR. Remove the #Gaggle header here.

Reply:
Resolving as can't fix in 3-6 months.